internal/workflow_testsuite.go

```diff
@@ -385,6 +390,11 @@ func (t *TestWorkflowEnvironment) SetWorkerOptions(options WorkerOptions) *TestW
 	return t
 }
 
+// SetWorkerStopChannel sets the worker stop channel to be returned from activity.GetWorkerShutdownChannel()
+func (t *TestWorkflowEnvironment) SetWorkerStopChannel(c chan struct{}) {
```

Comment:
Why do we need this? I thought worklows do not have access to it.

Reply:
Yes. Workflows do not require it. The reason why I added in workflow test suite is that the workflow test suite runs activities as well. To provide this method so that users can run activities with TestWorkflowEnvironment.

Reply:
I see. Then update the comment to explain how this channel should be used, please.